eventually/authentication/views.py

```diff
@@ -3,19 +3,66 @@
 ============
 """
 
-from utils.utils import json_loads
+
 from django.views.generic.base import View
 from django.contrib.auth import authenticate, login, logout
 from django.http import HttpResponse
+from eventually.settings import FRONT_HOST
+from utils.utils import json_loads
+from utils.jwttoken import create_token, handle_token
+from utils.send_mail import send_email
+from utils.validators import registration_validate
+from authentication.models import CustomUser
 
 
 class UserView(View):
     """UserView view handles GET, POST, PUT, DELETE requests."""
 
 
-def registration_user():
+def registration(request):
     """Registration CustomUser"""
 
+    if request.method == 'POST':
+        data = json_loads(request.body)
```

Comment:
maybe it makes sense to add the check for `None`?

Reply:
++